docs/source/examples.md

```diff
@@ -274,3 +274,87 @@ By default, the `hi-ml` package assumes that you supply a commandline argument `
 the commandline) to trigger a submission of the present script to AzureML. If you wish to control it via a different
 flag, coming out of your own argument parser, use the `submit_to_azureml` argument of the function
 `health.azure.himl.submit_to_azure_if_needed`. 
+
+## Training with k-fold cross validation in Azure ML
+To train with k-fold cross validation, you must do two things.
+
+1. Call the helper function `create_crossval_hyperdrive_config`
+to create an AML HyperDriveConfig object representing your parent run. It will have one child run for each of the k-fold
+splits you request, as follows
+    
+    ```python
+    from health_azure import create_crossval_hyperdrive_config
+    
+     hyperdrive_config = create_crossval_hyperdrive_config(num_cross_validation_splits,
+                                                           cross_val_split_name=cross_val_split_name,
+                                                           metric_name=metric_name)
+    ```
+    where:
+    - `num_cross_validation_splits` is the number of k-fold cross validation splits you require
+    - `cross_val_split_name` is the name of the argument given to each child run, whose value denotes which split that child represents (this parameter
```

Comment:
nitpicking - should we shorten the argument name? cross_val_count? The other one should more clealry say it's a commandline arg, maybe cross_val_index_arg_name?

Reply:
Sure - the function name already specified cross validation so maybe the arg name should be 'num_splits'?